group_vars/all

```diff
@@ -102,33 +103,34 @@ tarsnapper:
     schedule: hourly
     run_on: trusted
 
-gitannex:
-    stop_on_untrusted: True
+# no arm pkg, could use git-annex-git?
+#gitannex:
+#    stop_on_untrusted: True
 
 bitlbee:
-    run_on: trusted
-    torify: True
+    run_on: none
+    torify: False
 
-bootloader: grub
+bootloader: arm
 
 postgresql:
-    enable: True
+    enable: False
 
 fingerprint: False
 
 tor:
-    run_on: trusted
+    run_on: none
 
 localtime:
-    run_on: trusted
+    run_on: all
 
 nameservers:
     - 8.8.8.8       # Google
     - 209.244.0.3   # Level 3
     - 74.82.42.42   # Hurricane Electric
 
 bluetooth:
```

Comment:
If we just remove the bluetooth var entirely, nothing in the bluetooth role will be run. Is that preferable to switching `bluetooth.disable` here and then commenting out the rfkill stuff later?